servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/StacklessCancellationException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2024 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.concurrent.api;
+
+import io.servicetalk.concurrent.internal.ThrowableUtils;
+
+import java.util.concurrent.CancellationException;
+
+final class StacklessCancellationException extends CancellationException {
+    private static final long serialVersionUID = 5434529104526587317L;
```

Comment:
I think we need a new distinct version here, don't we?

Reply:
I always forget the rules. I do believe it is safe to change it either way so I will.